### PR TITLE
Add logo files and sources

### PR DESCRIPTION
**Pull request by planger**
Contributed on behalf of STMicroelectronics

### CODE REVIEW

**Review by sdirix — COMMENTED** [review]
The CDT Cloud logo itself is great.

Not so great is the Blueprint Logo. From afar the `I` is so thin that it's barely visible
image

The `R` looks super weird, as if the diagonal line was wrongly attached

image

Also the `INT` of `BLUEPRINT` seems to hover a bit

image

So I looked at the image in detail. The letters are not that greatly aligned which leads to the uneven impression.

image

Is there a way to get a better version of the Blueprint logo?

**Comment by planger** [comment]
Thank you for your feedback! Right, the BLUEPRINT font has indeed some flaws.
@sgraband Is it rather easy for you to incorporate Stefan's feedback above?

See also https://github.com/eclipsesource/cdt-cloud-blueprint/pull/3

**Comment by sgraband** [comment]
> Is there a way to get a better version of the Blueprint logo?

Tasking a professional to do it :wink:

Jokes aside, i can definately align the height of the letters. Not sure why this is messed up as i already once aligned them.
I can play around a bit with the alignment of the letters to see if it will look better.

Regarding the I and the R i am not sure how much i can improve this. The I is basically just a line with the width of the other letters. So i am not sure how to change this without breaking the font. Maybe the spacing will help? 
The problem with the R is that i needed to recreate it from another font to try and match the current font, as the original text did not have an R and the font is not detectable by inkscape. (Maybe this is possible in Illustrator). I can try to improve the diagonal line a bit.

**Comment by sgraband** [comment]
cdtcloud-blueprint

So i updated the things i just mentioned, could you take a look and give me feedback?

**Comment by sdirix** [comment]
Definitely better :+1:

**Comment by planger** [comment]
I agree, thanks! Looks much better.
Is there a reason why we wouldn't increase the size of the "BLUEPRINT" so it aligns with "CLOUD" above or align the B of BLUEPRINT with the L of CLOUD?

**Comment by sgraband** [comment]
Current:
Current
Aligned with C:
CAlign
Aligned with L:
LAlign
Same size centered:
SameSizeMiddle
Same size right align:
SameSizeLeft


I tried out some options here. Let me know what you think.
I personally still prefer the current solution, but i would also be fine with `Aligned with C` and `Same size centered` (Although i find this relatively bulky). Aligning with L i feel like is too symetric for my taste. WDYT?

**Comment by planger** [comment]
Yeah, you are right, it doesn't really improve.
I think I have a slight preference for the "aligned with L" variant or the current.
@sdirix WDYT?

**Comment by sdirix** [comment]
I would keep the current variant where it's not neatly aligned with the L. Seems a bit too tidy aligned with the L and reads like "LOUD BLUEPRINT"

File: logo/CDT_Cloud-short-black-on-white.svg
Comment (sdirix) [inline logo/CDT_Cloud-short-black-on-white.svg:1]: Any reason that the canvas is so large?

File: logo/CDT_Cloud_logo_BJ-01.jpg
Comment (sdirix) [inline logo/CDT_Cloud_logo_BJ-01.jpg:1]: same here

**Review by sdirix — APPROVED** [review]
LGTM. I have only nitpicky comments

File: logo/CDT_Cloud-short-black-on-white.svg
Comment (planger) [reply logo/CDT_Cloud-short-black-on-white.svg:1]: No real reason, I cropped the image.

File: logo/CDT_Cloud_logo_BJ-01.jpg
Comment (planger) [reply logo/CDT_Cloud_logo_BJ-01.jpg:1]: No real reason, I cropped the image.

**Comment by planger** [comment]
@MatthewKhouzam I see that you have added yourself as a reviewer to this PR. Do you want to add any feedback? Thanks!

**Pull request merged**